keyboards/keebio/quefrency/keymaps/yoryer/keymap.c

```diff
@@ -0,0 +1,47 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _BASE 0
+#define _FN1 1
+#define _RGB2 2
+
```

Comment:
You may want to consider switching this to an enum, as it may be easier to manage, long term.

However, this isn't a requirement, just a suggestion.

Reply:
Thank you for the suggestion, I've already made the update.